src/main/java/com/openshift/jenkins/plugins/pipeline/Argument.java

```diff
@@ -0,0 +1,28 @@
+package com.openshift.jenkins.plugins.pipeline;
+
+import hudson.Extension;
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+public class Argument extends AbstractDescribableImpl<Argument> {
+
+    protected final String value;
+
+    @DataBoundConstructor
+    public Argument(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    @Extension
+    public static class DescriptorImpl extends Descriptor<Argument> {
+        public String getDisplayName() {
+            return "Command argument";
+        }
```

Comment:
Can we do `FormValidation` style checking like we do in the build steps when extending Descriptor<Argument>?  If so, lets flag them if they add empty arguments.